examples/tensorflow/language-modeling/run_clm.py

```diff
@@ -148,13 +148,18 @@ class DataTrainingArguments:
             "help": "The percentage of the train set used as validation set in case there's no validation split"
         },
     )
-    max_seq_length: Optional[int] = field(
+    block_size: Optional[int] = field(
         default=None,
         metadata={
-            "help": "The maximum total input sequence length after tokenization. Sequences longer "
-            "than this will be truncated."
+            "help": "Optional input sequence length after tokenization. "
+            "The training dataset will be truncated in block of this size for training. "
+            "Default to the model max input length for single sentence inputs (take into account special tokens)."
         },
     )
+    pad_to_block_size: bool = field(
+        default=False,
+        metadata={"help": "Whether to pad all samples to `block_size`. "},
+    )
```

Comment:
```suggestion
```
Let's remove it entirely since it's not used.

Reply:
done!